ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```diff
@@ -15,7 +15,7 @@ base_images:
     name: centos
     namespace: openshift
     tag: stream9
-  ubi_minimal:
+  ubi-minimal:
```

Comment:
why "_" -> "-"?

Reply:
`could not load configuration from candidate revision of release repo: failed to load ci-operator configuration from release repo: invalid ci-operator config: invalid configuration: tests[12].test[0].from: 'ubi_minimal' is not a valid Kubernetes object name`